rmgpy/tools/extractInfoFromckcsv.py

```diff
@@ -192,4 +194,70 @@ def getROPFlux(spc_rop_dict, species_string, rxn_index):
             if rxnNum == rxn_index:
                 flux = flux_tup[1]
                 return flux
-    return []
\ No newline at end of file
+    return []
+
+def showTheMostDominantSpecies(dominant_list, moleFraction_dict, species_identifier_dict, displayMW=False):
+    """
+    find the most dominant species and display them in descending order
+    If displayMW is True, it also displays the molecular weight of each species
+    """
+    if len(dominant_list) is 1:
+        order = dominant_list[0]
+        v = list(moleFraction_dict.values())
+        k = list(moleFraction_dict.keys())
+        index_list = [i[0] for i in sorted(enumerate(v), key=lambda x:x[1], reverse=True)]
+        index_number = index_list[order]
+        species_dominant = k[index_number]
+        molecule_dominant = species_identifier_dict.get(species_dominant)
+        new_order = order+1
+        if new_order == 1:
+            print "****The most dominant species****"
+        elif new_order == 2:
+            print "****The 2nd most dominant species****"
+        elif new_order == 3:
+            print "****The 3rd most dominant species****"
+        elif new_order % 10 is 1 and new_order is not 11:
+            print "****The {0}st most dominant species****".format(new_order)
+        elif new_order % 10 is 2 and new_order is not 12:
+            print "****The {0}nd most dominant species****".format(new_order)
+        elif new_order % 10 is 3 and new_order is not 13:
+            print "****The {0}rd most dominant species****".format(new_order)
```

Comment:
instead of printing a long log, can you convert it into a table in the same format as other functionalities adopt?